composer.json

```
@@ -34,7 +34,8 @@
     "autoload": {
         "psr-4": {
             "EzSystems\\EzRecommendationClientBundle\\": "src/bundle/",
-            "EzSystems\\EzRecommendationClient\\": "src/lib/"
+            "EzSystems\\EzRecommendationClient\\": "src/lib/",
+            "Ibexa\\Contracts\\PersonalizationClient\\": "src/contracts/"
```

I would rather see this as:
```suggestion
            "Ibexa\\Contracts\\Personalization\\Client\\": "src/contracts/"
```
@adamwojs @Steveb-p WDYT?

---

I'm ok with either. The second can introduce class name collisions if there would be a package with `Ibexa\Contracts\Personalization` classes that adds `Client` sub-namespace, but I think we can manage that.

:+1:

---

This repository will be moved to ezplatform-personalization (ibexa/personalization) so maybe better idea is to add just `"Ibexa\\Contracts\\Personalization\\": "src/contracts/"`?